www-cronrobot/app/views/home/index.html.erb

```diff
@@ -1 +1,176 @@
-welcome to CronRobot
\ No newline at end of file
+<section id="home">
+
+    <!-- The Navbar of the website, will contain the logo, and the links -->
+    <nav>
+      <a href="#home">
+        <img id="logo" alt="Logo" src="images/Big Logo.svg">
+      </a>
+      <div id="center-nav">
+        <a href="#features">Features</a>
+        <a href="#pricing">Pricing</a>
+        <a href="#faq">FAQ</a>
+        <a href="https://docs.cronrobot.io">Docs</a>
+      </div>
+      <a onclick="handleLogin()" class="center" id="dashboard">
+        <img src="images/icons/dashboard.svg">
+        <p class="text-medium">Dashboard</p>
+      </a>
+      <button class="openbtn" onclick="openNav()">☰</button>
+    </nav>
+
+    <h1 id="heading">
+      The world's leading uptime monitoring service<blue>.</blue>
```

Comment:
what if we change to green?

blue should be a color variable, not a tag IMO

Reply:
We could change it to the green color used below, as for the tag, I wasn't sure how to only change the color of one letter of a text, because using nested HTML tags was making it a separate line, would appreciate advice on this.

Reply:
probably with something like: <span with some class>.</span>

Reply:
Done 👍